File: c/1-sequence-struct/subtraction/main.c
// Write a program that asks for two integers and prints the subtraction.

#include <stdio.h>
#include <stdlib.h>

int main()
{
    int n1, n2, sub;

    printf("Enter the first number: ");
    scanf("%d", &n1);
    printf("Enter the second number: ");
    scanf("%d", &n2);

    sub = n1 - n2;
    printf("\nSubtraction of the entered numbers is %d.\n", sub);
    return 0;
}

File: java/1-sequence-struct/subtraction/src/Main.java
// Write a program that asks for two integers and prints the subtraction.

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        // Instantiating object in java for data entry
        Scanner sc = new Scanner(System.in);

        int n1, n2, sub;

        System.out.print("Enter the first number: ");
        n1 = sc.nextInt();
        System.out.print("Enter the second number: ");
        n2 = sc.nextInt();

        sub = n1 - n2;
        System.out.println("\nSubtraction of the entered numbers is " + sub);
    }
}
